lyrics/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from lyrics.models  import Lyric
+
+
+class LyricSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Lyric
+        fields = ["title","description","content","composer"]
```

Comment:
where's the `author` field?
Also I've modified the tests for serializers. 
1. Switch to master and pull latest changes
2. Switch back to your working branch and merge master with: `git merge master`
3. Add author field to the serializers and finally make your PR again.

Reply:
okay will do that , but when i add the author it was also not passing the test all the same will do

Reply:
pull upstream master into the master branch and merge master into the branch you're in before you make the changes.
Check the above comments.